app/jobs/events/bust_cache_job.rb

```diff
@@ -0,0 +1,9 @@
+module Events
+  class BustCacheJob < ApplicationJob
+    queue_as :events_bust_cache
+
+    def perform(cache_buster = CacheBuster.new)
+      cache_buster.bust_event
```

Comment:
Maybe `bust_events` since it busts the whole `/events` page?

Reply:
Yes, I have updated the name of the method.